CONTRIBUTING.md

```diff
@@ -52,27 +52,42 @@ gem server, please consider checking out
 provide pass-through caching for RubyGems.org, as well as host private
 gems for your organization..**
 
+### Getting the code
+
+Clone the repo: `git clone git@github.com:rubygems/rubygems.org.git`
```

Comment:
I think we need public git url here aka `https://github.com/rubygems/rubygems.org.git`. This one is for members only if I understand it well.

Reply:
Hmm, it seems just copied, was it wrong for whole time? Does it work with no access to rubygems.org? 🤔

Reply:
I updated to the https url. I assume most people that want the git+ssh scheme will know how to use it, and they probably found the repository and cloned it from github before they even read this document. I suspect this instruction is more for completeness than any actual necessary instruction.